src/scss/6-elements/_input-button.scss

```diff
@@ -57,43 +57,37 @@
   #{$theme-dark} &  { @include input-dark-mode(); }
   #{$theme-light} & { @include input-light-mode(); }
 
-  color: hsl(var(--p-input-text-color));
-  background-color: hsl(var(--p-input-bg-color));
-  border: solid pxToRem(1) hsl(var(--p-input-border-color));
+  color: hsl(var(--p-input-text-color)) !important;
+  background-color: hsl(var(--p-input-bg-color)) !important;
+  border: solid pxToRem(1) hsl(var(--p-input-border-color)) !important;
```

Comment:
this seems annoying, could we fix those with layers? or will it be a bigger mess then?

Reply:
Was a fix from a bit ago that isn't necessary anymore. Reverted.